operator/go.mod

```diff
@@ -3,26 +3,26 @@ module github.com/grafana/loki/operator
 go 1.17
 
 require (
-	github.com/ViaQ/logerr v1.0.10
-	github.com/go-logr/logr v0.4.0
+	github.com/ViaQ/logerr v1.1.0
+	github.com/go-logr/logr v1.2.3
 	github.com/google/uuid v1.1.2
 	github.com/imdario/mergo v0.3.12
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.3.0
 	github.com/openshift/api v0.0.0-20210901140736-d8ed1449662d // release-4.9
```

Comment:
This needs also a bump for using the 1.23-based openshift api `go get github.com/openshift/api@release-4.10`